src/main/steps/applicant1/confirm-your-joint-application/content.ts

```diff
@@ -91,8 +92,25 @@ const en = ({ isDivorce, partner, userCase, userEmail, isApplicant2 }: CommonCon
   subHeading5: `Reason for  ${isDivorce ? 'the divorce' : 'ending the civil partnership'}`,
   line18: `The ${isDivorce ? 'marriage' : 'relationship'} has broken down irretrievably (it cannot be saved).`,
   subHeading6: 'Financial order application',
-  applicant1FinancialOrderYes: 'Applicant 1 is applying to the court for financial orders',
-  applicant2FinancialOrderYes: 'Applicant 2 is applying to the court for financial orders',
+  applicant1FinancialOrderYes: `Applicant 1 is intending to apply to the court for financial orders ${
+    userCase.whoIsFinancialOrderFor?.includes(FinancialOrderFor.APPLICANT) ? 'for the applicant' : ''
+  }${
+    isEqual(userCase.whoIsFinancialOrderFor, [FinancialOrderFor.APPLICANT, FinancialOrderFor.CHILDREN]) ? ' and ' : ''
+  }${
+    userCase.whoIsFinancialOrderFor?.includes(FinancialOrderFor.CHILDREN)
+      ? 'for the children of the applicant and the respondent.'
+      : '.'
+  }`,
```

Comment:
haven't tried so not 100% sure but we could do 

`userCase.whoIsFinancialOrderFor?.join(' and ').replace(FinancialOrderFor.APPLICANT, 'for the applicant').replace(FinancialOrderFor.CHILDREN, 'for the children of the applicant and the respondent')`

Reply:
also this is the confirm-your-joint-application page so I assume 'respondent' should be 'applicant 2' and 'applicant' should be 'applicant 1'

Reply:
that method is sooo much better, nice one.